homeassistant/components/binary_sensor/tellduslive.py

```diff
@@ -9,22 +9,36 @@
 """
 import logging
 
-from homeassistant.components import tellduslive
+from homeassistant.components import binary_sensor, tellduslive
 from homeassistant.components.binary_sensor import BinarySensorDevice
 from homeassistant.components.tellduslive.entry import TelldusLiveEntity
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
 _LOGGER = logging.getLogger(__name__)
 
+DOMAIN = tellduslive.DOMAIN
```

Comment:
Please remove this. We're not allowed to overwrite the platform domain with some other domain. This platform belongs to the binary_sensor domain.

Reply:
I added this as I got the following error:

```log
2018-12-09 20:16:11 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/Users/fer/Documents/github/home-assistant/homeassistant/config_entries.py", line 468, in async_forward_entry_unload
    self.hass, component=getattr(self.hass.components, component))
  File "/Users/fer/Documents/github/home-assistant/homeassistant/config_entries.py", line 294, in async_unload
    if component.DOMAIN == self.domain:
  File "/Users/fer/Documents/github/home-assistant/homeassistant/loader.py", line 154, in __getattr__
    value = getattr(self._module, attr)
AttributeError: module 'homeassistant.components.binary_sensor.tellduslive' has no attribute 'DOMAIN'
```

Reply:
Is this better?

```python
DOMAIN = '{}_{}'.format(binary_sensor.DOMAIN, tellduslive.DOMAIN)
```

Reply:
It shouldn't be needed. Something else is wrong. Probably with how we unload the entries.